dali/test/python/test_pipeline.py

```diff
@@ -784,6 +784,64 @@ def __next__(self):
             self.i = (self.i + 1) % self.n
         return (pos, size)
     next = __next__
+    
+def test_lazy_init():
+    """
+        Comparing results of pipeline: lazy_init false and lazy_init true with empty folder and real folder
+    """
+    batch_size =128
+
+    class LazyPipeline(Pipeline):
+        def __init__(self, batch_size, num_threads, device_id, num_gpus, db_folder, lazy_type):
+            super(LazyPipeline, self).__init__(batch_size,
+                                             num_threads,
+                                             device_id)
+            self.input = ops.CaffeReader(path = db_folder, shard_id = device_id, num_shards = num_gpus, lazy_init = lazy_type)
+            self.decode = ops.nvJPEGDecoder(device = "mixed", output_type = types.RGB)
+            self.pos_rng_x = ops.Uniform(range = (0.0, 1.0), seed=1234)
+            self.pos_rng_y = ops.Uniform(range = (0.0, 1.0), seed=5678)
+            self.crop = ops.Crop(device="gpu", crop =(224,224))
+
+        def define_graph(self):
+            self.jpegs, self.labels = self.input()
+
+            pos_x = self.pos_rng_x()
+            pos_y = self.pos_rng_y()
+            images = self.decode(self.jpegs)
+            crop = self.crop(images, crop_pos_x=pos_x, crop_pos_y=pos_y)
+            return (crop, self.labels)
+
+
+        def iter_setup(self):
```

Comment:
You can remove it.

Reply:
HI, Janusz, do you mean from L812 to L815 could be removed?

Reply:
As it is empty anyway you can safely remove `iter_setup`.